src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -58,19 +58,19 @@ public void testDot()
         checkConfig.addAttribute("option", "NL");
         checkConfig.addAttribute("tokens", "DOT");
         final String[] expected = {
-            "31:10: " + getCheckMessage(MSG_LINE_NEW, "."),
+            "29:10: " + getCheckMessage(MSG_LINE_NEW, "."),
```

Comment:
Why did line number change if input was copied? We aren't expecting the input to change except for package.